test.js

```diff
@@ -57,6 +57,14 @@ test('removeQueryParameters option', t => {
 	t.is(m('www.sindresorhus.com?foo=bar&utm_medium=test&ref=test_ref', opts), 'http://www.sindresorhus.com/?foo=bar');
 });
 
+test('normalizeHttps option', t => {
+	const opts = {normalizeHttps: true};
+
+	t.is(m('http://sindresorhus.com'), 'http://sindresorhus.com');
```

Comment:
@shinnn No, more to show that if the option is not given, nothing changes. I'll updated this to make more sense.